tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,122 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Plex
+finish-args:
+  - --socket=x11
+  # currently not working - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-cache:rw
+  - --filesystem=xdg-config:rw
+  - --filesystem=xdg-data:rw
```

Comment:
```suggestion
```

At least in theory, these three shouldn't be needed. Flatpak exports `$XDG_CONFIG_HOME`, `$XDG_CACHE_HOME` and `$XDG_DATA_HOME` environment variables that are writable by default, and it would be the best to make the app respect these if they aren't. If not, please limit these to specific directories needed by Plex. e.g. `--filesystem=xdg-config/plex` (just an example as I did not check it!(

Reply:
I've just checked and it seems to properly use all three variables by default.

Reply:
Correct, it respects these variables. But what are the permissions for then?

Reply:
They can be dropped altogether. These permissions are for vendors who want their apps to keep using existing configuration outside the sandbox, or access files belonging to other programs (e.g. mod manager for a game).